securedrop_client/logic.py

```diff
@@ -970,6 +991,15 @@ def on_file_download_failure(self, exception: Exception) -> None:
                 self.file_missing.emit(f.source.uuid, f.uuid, str(f))
             self.gui.update_error_status(_("The file download failed. Please try again."))
 
+    def on_delete_conversation_success(self, uuid: str) -> None:
+        logger.info("Conversation %s successfully deleted at server", uuid)
```

Comment:
here and in `on_delete_source_success` is where you would trigger a sync, but you mentioned not wanting to do that yet. For clarification, do you plan to add this behavior in a follow-up PR?

Reply:
I was going to add that here after we'd otherwise verified the UX, but it could be done separately.

Reply:
Let's add this in a followup PR that anyone can review while I'm afk for break to prioritize getting all the UI changes in before I leave